### PR TITLE
Feature/fetch api

### LINKED ISSUE #17: Add Fetch
As a dev I would like to pull data from the server

### PR DESCRIPTION
**Pull request by cameronRomo**
## What is the change?
* App now fetches data

## What does it fix? 
* Remove hard data
* Refactor classes to not have floats

## Is this a fix or a feature?  
* Feature and some fixes

## Where should the reviewer start?
* Biggest change `scripts.js`:
  * line 24
* Class changes:
  * `Activity.js`
    * line 12
  * `Hydration.js`
    * line 15

## How should this be tested?
* Pulldown
  * Run `npm test`
  * Run `npm start`

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -21,6 +21,39 @@ let hydrationRepo;
...
+  .then(data => data.activityData)
+  .catch(error => console.log(error));
+
+const fetchedHydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData')
+  .then(response => response.json())
+  .then(data => data.hydrationData)
+  .catch(error => console.log(error));
+
+
+Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydrationData])
```
Comment (mdflynn) [inline src/scripts.js:45]: Nice way to resolve the promises! Great job getting the server data to update.

File: src/scripts.js [hunk 2]
```diff
@@ -63,7 +96,8 @@ function createUserData() {
 }
 
 function makeUsers(array) {
-  userData.forEach(function(dataItem) {
+  console.log(userData);
```
Comment (mdflynn) [inline src/scripts.js]: Remove this console.log()

File: src/scripts.js [hunk 3]
```diff
@@ -21,6 +25,39 @@ let hydrationRepo;
 let sleepRepo;
 let activityRepo;
 
+const fetchedUserData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
```
Comment (okalimbet) [inline src/scripts.js:24]: Nice user of fetch to get data from the server!

**Review by mdflynn — APPROVED** [review]
Great PR. It's good to see the server data displayed on the page.

File: src/Activity.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ class Activity {
 
   getMilesFromStepsByDate(id, date, userRepo) {
     let userStepsByDate = this.getUserInfoByDateAndId(id, date);
-    return parseFloat(((userStepsByDate.numSteps * userRepo.strideLength) / 5280).toFixed(1));
+    return parseFloat(((userStepsByDate.numSteps * userRepo.strideLength) / 5280).toFixed(0));
```
Comment (okalimbet) [inline src/Activity.js:12]: good catch on decimal issue and using toFixed to solve it!

File: src/scripts.js (hunk 2 above)
Comment (okalimbet) [inline src/scripts.js]: We need to remove this console.log()

**Review by okalimbet — CHANGES_REQUESTED** [review]

**Review by okalimbet — APPROVED** [review]

File: src/scripts.js (hunk 3 above)
Comment (cameronRomo) [reply src/scripts.js:24]: Log gone!

**Pull request merged**